recipes/kealib/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd trunk
+cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
+-D CMAKE_SKIP_RPATH=ON \
+-D HDF5_INCLUDE_DIR=$PREFIX/include \
+-D HDF5_LIB_PATH=$PREFIX/lib \
+.
+
+make
+make install
```

Comment:
No, but I've created a kealib issue for this: https://bitbucket.org/chchrsc/kealib/issues/1/add-make-check-target. Will do this for the next release.